server/proxy/proxy.go

```diff
@@ -116,6 +117,10 @@ func (p *Proxy) Notify(cc *apps.Context, subj apps.Subject) error {
 		return err
 	}
 
+	return p.notify(cc, subs)
+}
+
+func (p *Proxy) notify(cc *apps.Context, subs []*apps.Subscription) error {
 	expander := p.newExpander(cc, p.mm, p.conf, p.store, "")
 
 	notify := func(sub *apps.Subscription) error {
```

Comment:
This may be considered as shadowing the function. Does it make sense to use a different name?